packages/react/src/ButtonGroup/ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import {ComponentMeta, ComponentStory} from '@storybook/react'
+import React from 'react'
+import {ButtonGroup} from '.'
+import {Button} from '../Button'
+
+export default {
+  title: 'Components/ButtonGroup',
+  component: ButtonGroup
+} as ComponentMeta<typeof ButtonGroup>
+
+const Template: ComponentStory<typeof ButtonGroup> = args => (
+  <ButtonGroup {...args}>
+    <Button>This is one button</Button>
+    <Button>This is the second button</Button>
+  </ButtonGroup>
+)
+
+export const Primary = Template.bind({})
+Primary.args = {}
+
+const SingleButtonTemplate: ComponentStory<typeof ButtonGroup> = args => (
+  <ButtonGroup {...args}>
+    <Button>This is one button</Button>
+  </ButtonGroup>
+)
+
+export const SingleButton = SingleButtonTemplate.bind({})
+
+const LargeButtonTemplate: ComponentStory<typeof ButtonGroup> = args => (
+  <ButtonGroup buttonSize="large" {...args}>
+    <Button>This is one button</Button>
+    <Button>This is the second button</Button>
+  </ButtonGroup>
+)
+
+export const LargeButtons = LargeButtonTemplate.bind({})
+
+const VerticalButtonTemplate: ComponentStory<typeof ButtonGroup> = args => (
+  <ButtonGroup direction="vertical" {...args}>
+    <Button>This is one button</Button>
+    <Button>This is the second button</Button>
+  </ButtonGroup>
+)
+
+export const VerticalButtons = VerticalButtonTemplate.bind({})
+
+const LinkButtonTemplate: ComponentStory<typeof ButtonGroup> = args => (
+  <ButtonGroup buttonsAs={'a'} {...args}>
```

Comment:
This one is missing a `href`, which should have been a required prop. Wonder why type checker didn't pick it up 🤔 

```suggestion
  <ButtonGroup buttonsAs={'a'} href="#" {...args}>
```

Reply:
I will add this but it should likely but checked in the main component

Reply:
And it wouldn't go on the group but on the button itself.

Reply:
ah yes, good point.